server/game/cards/08.6-SAT/TheKingOnTheWall.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../drawcard.js');
+
+class TheKingOnTheWall extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction({ faction: 'targaryen', unique: true });
+        this.whileAttached({
+            effect: ability.effects.addTrait('King')
+        });
+
+        this.reaction({
+            when: {
+                afterChallenge: event =>
+                    event.challenge.winner === this.controller &&
+                    event.challenge.challengeType === 'power' &&
+                    event.challenge.isParticipating(this.parent) &&
+                    this.numberOfOutOfFactionLocations() > 0 &&
+                    this.controller.canDraw()
+            },
+            handler: () => {
+                var cards = this.numberOfOutOfFactionLocations();
```

Comment:
var --> let

Reply:
nope, as it is overwritten next line to retrieve the number of *actual* cards drawn

Reply:
It's still in the same block though so that should be fine? Not that it matters a lot.

Reply:
fixed